Source/WebCore/html/NumberInputType.cpp

```diff
@@ -141,6 +143,110 @@ bool NumberInputType::typeMismatch() const
     return false;
 }
 
+ValueOrReference<String> NumberInputType::stripInvalidNumberCharacters(const String& input)
+{
+    const auto allowedChars = StringView::fromLatin1("0123456789.Ee-+");
+    const size_t len = input.length();
```

Comment:
```suggestion
    const auto length = input.length();
```
(style guide generally recommends using full names when possible)